operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -274,6 +274,23 @@ private static void processServerEvent(CoreV1Event event) {
           .ifPresent(info -> info.updateLastKnownServerStatus(serverName, status));
   }
 
+  /**
+   * Get all the domain resources in the given namespace.
+   *
+   * @param ns the namespace
+   * @return list of the domain resources
+   */
+  public static List<DomainResource> getDomains(String ns) {
+    List<DomainResource> referencingDomains = new ArrayList<>();
```

Comment:
minor comment: should variable name be domains instead of referencingDomains since we are getting all domains within the namespace ns?

Reply:
good eye. I started with a method that returns only domains that reference a given cluster, and later changed to return all. will change. thanks.